### ISSUE TITLE
upgrade dor-services so that rights indexing is done correctly

### ISSUE THREAD

**Issue by jmartin-sul** [sul-dlss/dor_indexing_app#25]
rights indexing has changed slightly with the addition of location-based rights.  dor-services has been fixed to accommodate this, and apps that use dor-services to index things should be upgraded accordingly.

**Comment by drh-stanford** [comment]
can you be more specific about the required `dor-services` version needed? as of 11/3 we're on 5.12.0

**Comment by jmartin-sul** [comment]
i think we want v5.14.0:
https://github.com/sul-dlss/dor-services/releases/tag/v5.14.0
https://github.com/sul-dlss/dor-services/compare/v5.12.0...v5.14.0

**Comment by drh-stanford** [comment]
ok. do we need to reindex any objects as part of the upgrade to 5.14.0 in production?

**Comment by jmartin-sul** [comment]
i don't think so, since i don't think we actually expose that index field anywhere.  if we did decide to use that index field, we'd have to reindex:
* objects with location based rights
* objects with the `no-download` rule
* (optional) "dark" and "citation only" objects (the text is slightly different, old code would've indexed `Dark` and `None`, respectively; new code would index `Dark (Preserve Only)` and `Citation Only`).

so, not an urgent upgrade, but the right thing to do in the not too distant future.

we may want to hold off until we've got a satisfactory handle on sul-dlss/argo#865, since the latest dor-services seems to have problems indexing objects that don't have an active fedora model (including items that belong to collections without an active fedora model).
